File: backend/Services/Tours/Tours.Application/UseCases/Tours/Commands/DTOs/AddTourDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Tours.Application.UseCases.Tours.Commands.DTOs
{
    public class AddTourDTO
    {
        public string Name { get; set; }
        public Guid MountainId { get; set; }
        public string Description { get; set; }
        public int MinNumberOfPeople { get; init; }
        public int MaxNumberOfPeople { get; init; }
        public DateTime Date { get; set; }
    }
}

File: frontend/src/features/tour/types/AddTourDto.ts
export interface AddTourDto {
  name: string;
  mountainId: string;
  description: string;
  minNumberOfPeople: number;
  maxNumberOfPeople: number;
  date: string;
}

export interface AddTourCommand {
  addTourDTO: AddTourDto;
  createdBy: string;
}
